crud/update.lua

```diff
@@ -24,10 +24,32 @@ local function update_on_storage(space_name, key, operations, field_names)
 
     -- add_space_schema_hash is false because
     -- reloading space format on router can't avoid update error on storage
-    return schema.wrap_box_space_func_result(space, 'update', {key, operations}, {
+    local res, err = schema.wrap_box_space_func_result(space, 'update', {key, operations}, {
         add_space_schema_hash = false,
         field_names = field_names,
     })
+
+    if err ~= nil then
+        return nil, err
+    end
+
+    if res.err == nil then
+        return res, nil
+    end
+
+    -- We can only add fields to end of the tuple.
+    -- If schema is updated and intermediate null fields are added, then we will get error.
```

Comment:
`null` -> `nullable`

 `intermediate` ... Not sure it's correct. You can only append fields to format. They are trailing.